OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -498,11 +538,28 @@ function Test-TargetResource {
         }
     }
 
+    # Check to see if Spaces array is empty
```

Comment:
Incorrect comment - its not an array.
Both this and the next comment are a bit redundant